Makefile

```
@@ -29,3 +29,20 @@ install:
 
 install-dev:
 	pip install -r requirements-dev.txt
+
+package:
+	python3 setup.py sdist bdist_wheel
+
+clean: clean-build clean-pyc
+
+clean-build:
+	rm -fr build/
+	rm -fr dist/
+	rm -fr .eggs/
+	find . -name '*.egg-info' -exec rm -fr {} +
+	find . -name '*.egg' -exec rm -f {} +
+
+clean-pyc:
+	find . -name '*.pyc' -exec rm -f {} +
```

```suggestion
	find . -name '*.pyc' -delete
```
If that works on OSX, too. That might be a GNU find extension.

---

Seems to work fine